kornia/enhance/adjust.py

```diff
@@ -343,7 +343,7 @@ def adjust_contrast(image: Tensor, factor: Union[float, Tensor], clip_output: bo
     while len(factor.shape) != len(image.shape):
         factor = factor[..., None]
 
-    KORNIA_CHECK(any(factor >= 0), f"Contrast factor must be positive. Got {factor}")
+    KORNIA_CHECK(any(factor >= 0), "Contrast factor must be positive.")
```

Comment:
Wouldn't this be better too?
```suggestion
    KORNIA_CHECK((factor >= 0).any(), "Contrast factor must be positive.")
```